src/test/java/core/basesyntax/dao/StorageDaoImplTest.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax.dao;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.model.User;
+import java.time.LocalDateTime;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class StorageDaoImplTest {

```

Comment:
we do not really need to test storage in this task, but since you already did - good job! :+1: